### ISSUE TITLE
error can't covert string to integer

### ISSUE THREAD

**Issue by RyanZhu1024** [Linuxbrew/linuxbrew#598]
run `brew postinstall glibc`

```
Error: can't convert String into Integer

/usr/lib/ruby/1.8/fileutils.rb:1223:in `chmod'
/usr/lib/ruby/1.8/fileutils.rb:1223:in `chmod'
/usr/lib/ruby/1.8/fileutils.rb:876:in `chmod'
/usr/lib/ruby/1.8/fileutils.rb:875:in `each'
/usr/lib/ruby/1.8/fileutils.rb:875:in `chmod'
/home4/xzhu20/.linuxbrew/Library/Formula/glibc.rb:35:in `post_install'
/home4/xzhu20/.linuxbrew/Library/Homebrew/formula.rb:750:in `run_post_install'
/home4/xzhu20/.linuxbrew/Library/Homebrew/postinstall.rb:16
```

**Comment by vishgala** [comment]
Facing the same issue. Any updates ?

**Comment by dunn** [comment]
I don't think `"+x"` is a valid argument for Ruby's version of `chmod`.  Try changing line 35 to something like 
```ruby
    chmod 0755, [lib/"ld-#{version}.so", lib/"libc-#{version}.so"]

**Comment by sjackman** [comment]
Or upgrade your version of Ruby if possible.

**Comment by sjackman** [comment]
Resolved by #808, thanks to @maxim-belkin.
Ruby 1.8 has only integer mode chmod. Ruby 1.9 introduced string mode.
